docs/api-reference/react/deckgl.md

```diff
@@ -47,6 +47,15 @@ const App = (data) => (
 
 `DeckGL` accepts all [Deck](/docs/api-reference/deck.md#properties) properties, with these additional semantics:
 
+### React Context
+
+##### `ContextProvider` (React.Component, optional)
```

Comment:
Do you think should mention react-map-gl MapContext here?

Reply:
I can put an example snippet here, but this feature is not limited to using react-map-gl.